app/src/main/java/com/github/palFinderTeam/palfinder/navigation/MainNavActivity.kt

```diff
@@ -172,10 +184,35 @@ class MainNavActivity : AppCompatActivity() {
                 getString(R.string.scanned)+ ": " + result.contents,
                 Toast.LENGTH_LONG
             ).show()
-            createPopUp(this, {
-                //follow user or join meetup
-            }, textId = R.string.qr_scan_follow_account,
-                continueButtonTextId = R.string.follow)
+            if (result.contents.startsWith(USER_ID)) {
```

Comment:
This is indeed better than the 0 and 1 for meetups/profile QR codes, glad you changed it!